app/models/ext_management_system.rb

```diff
@@ -414,6 +416,14 @@ def self.ems_infra_discovery_types
     @ems_infra_discovery_types ||= %w(virtualcenter scvmm rhevm)
   end
 
+  def disable_ems
+    update_attributes(:enabled => false)
```

Comment:
oh, scratch that. I was reading `update_attribute` :/

But I think it should be `update_attributes!`